app/views/concerns/facet_entry_point_displaying_view.rb

```diff
@@ -49,4 +49,11 @@ def facet_entry_field_title(facet_link_group)
     presenter = FacetPresenter.build(ff, controller)
     presenter.facet_title || facet_link_group.facet_field
   end
+
+  def facet_entry_label(entry)
```

Comment:
I think more specific methods for facet entries in the FacetPresenter make sense, in fact I'm thinking now it might make sense to add a FacetLinkGroupPresenter and have the things that are shared between this new presenter and the existing FacetPresenter moved to a concern under presenters.